doc/release/1.8.0-notes.rst

```diff
@@ -137,7 +139,9 @@ nonsingular non-Hermitian linear systems in `scipy.sparse.linalg.tfqmr`.
 
 The sparse SVD library PROPACK is now vendored with SciPy, and an interface is
 exposed via `scipy.sparse.svds` with ``solver='PROPACK'``. For some problems,
-this may be faster and/or more accurate than the default, ARPACK.
+this may be faster and/or more accurate than the default, ARPACK. PROPACK
+functionality is currently opt-in--you must specify ``USE_PROPACK=1`` at
+source build time to use it.
```

Comment:
Add "due to potential issues on Windows that we aim to resolve in the next release".